File: semestre-1/algoritmos/lista-1/python/MR-Alg-01-Ex-14.py
print("Programa para calcular a área de um triângulo, baseado no comprimento dos lados")
l1 = float(input("Lado 1 (em cm): "))
l2 = float(input("Lado 2 (em cm): "))
l3 = float(input("Lado 3 (em cm): "))

l = (l1 + l2 + l3)/2

area = (l * (l - l1) * (l - l2) * (l - l3))**.5

print(f"Área do triângulo: {area} cm²")

File: semestre-1/algoritmos/lista-1/c/MR-Alg-01-Ex-14.c
#include <stdio.h>
#include <math.h>

void main() {
    float l1, l2, l3;

    printf("Programa para calcular a área de um triângulo, baseado no comprimento dos lados.\n");
    printf("Lado 1 (em cm): ");
    scanf("%f", &l1);

    printf("Lado 2 (em cm): ");
    scanf("%f", &l2);

    printf("Lado 3 (em cm): ");
    scanf("%f", &l3);

    float l = (l1 + l2 + l3) / 2;
    float area = powf(l * (l - l1) * (l - l2) * (l - l3), .5);

    printf("\nÁrea do triângulo: %.2f cm²\n", area);
}